<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1584</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>난폭운전방지</string>
  </property>
  <widget class="QTabWidget" name="Controll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1542</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab { height: 40; width: 514px; font-size: 12pt; }</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Main">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>1171</width>
       <height>751</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Connecting to server...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_logout">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>60</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lastPenalty">
     <property name="geometry">
      <rect>
       <x>1370</x>
       <y>380</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Color : red</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>150</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_car_number">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>plate_num</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_plate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>License Plate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>1230</x>
       <y>260</y>
       <width>281</width>
       <height>521</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_score">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>score</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD_score">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLCDNumber{ color: rgb(0, 220, 0); }</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>speed</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD_speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>status</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status_desc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>211</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Log">
    <attribute name="title">
     <string>Penalty History</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1471</width>
       <height>711</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>9</number>
     </property>
     <column>
      <property name="text">
       <string>datetime</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>minus reason</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>minus score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>speed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>image_path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>image_name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>json_data</string>
      </property>
     </column>
    </widget>
    <widget class="QDateTimeEdit" name="dateTime_start_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTime_end_1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_1">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string> ~ </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_1">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>minus reason</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>penalty</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>speed</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_preview_1">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>861</width>
       <height>691</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Image Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="object_log">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Admin Log</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>1471</width>
       <height>711</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <column>
      <property name="text">
       <string>datetime</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>plate number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>object</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>image_path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>iamge_name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>json_data</string>
      </property>
     </column>
    </widget>
    <widget class="QDateTimeEdit" name="dateTime_start_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yy. M. d.  AP h:mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string> ~ </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_search_2">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTime_end_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_preview_2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>90</y>
       <width>861</width>
       <height>691</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Image Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>plate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>object</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
